client/components/domains/contact-details-form-fields/README.md

```diff
@@ -95,7 +95,7 @@ Example:
 {
     submitButton: this.hasAnotherStep()
         ? translate( 'Continue' )
-        : translate( 'Continue to Checkout' ),
+        : translate( 'Continue' ),
```

Comment:
Maybe we could remove the duplicate `translate( 'Continue' ),` here and just write:

```
   submitButton:  translate( 'Continue' ),
```

Reply:
Good catch, Ramon. I did make a similar change here 7891a3aa7592dd2ad8753ee04ec216c363333f58 but missed updating the docs. I have updated now with 76521bebf0ead2e4cdfec75d75e786bdbc69519b.